### PR TITLE
remove the deprecated comment

### PR DESCRIPTION
**Pull request by jianglingxia**
**What this PR does / why we need it**:
remove the deprecated comment
**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by jianglingxia** [comment]
/assign @dchen1107  @mikedanese 
thanks

File: cmd/kube-proxy/app/server.go [hunk 1]
```diff
@@ -99,12 +99,6 @@ type Options struct {
 	CleanupIPVS bool
 	// config is the proxy server's configuration object.
 	config *kubeproxyconfig.KubeProxyConfiguration
-
-	// The fields below here are placeholders for flags that can't be directly mapped into
```
Comment (dixudx) [inline cmd/kube-proxy/app/server.go]: Why removing this? Have the deprecated flags already been removed?

I don't think this is right.

/cc @ncdc PTAL.

**Review by dixudx — CHANGES_REQUESTED** [review]
It's too early to remove this comment.

File: cmd/kube-proxy/app/server.go (hunk 1 above)
Comment (ncdc) [reply cmd/kube-proxy/app/server.go]: Correct, we can't do this yet

**Comment by dchen1107** [comment]
It is too early to do this. Close the pr. Thanks!

**Pull request closed without merging**